app.js

```diff
@@ -25,6 +25,12 @@ app.use(parser.json());
 var router = require('./server/routes');
 app.use(router);
 
+
```

Comment:
We don't need this addition do we? Seems like a relic of merge sometime in the past